presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -1199,7 +1199,7 @@ public void testListUnknownSchema()
     }
 
     @Test
-    public void testGetPartitions()
+    public void testGetPartitions() throws Exception
```

Comment:
put `throws ..` to its own line with 8 spaces for indentation.

Reply:
Sure